### PR TITLE
Add fitted components

### PR DESCRIPTION
**Pull request by FadhlanR**
https://github.com/user-attachments/assets/9dfb5729-d980-4429-b7b5-724471b457d3

https://github.com/user-attachments/assets/cd8c34b8-f5e8-49fc-af45-945ce73ab2b6

https://github.com/user-attachments/assets/9bdfe060-9b06-4283-a9cf-945b741ce783

### CODE REVIEW

File: packages/experiments-realm/person.gts [hunk 1]
```diff
@@ -12,6 +12,102 @@ import { GridContainer } from '@cardstack/boxel-ui/components';
...
+        justify-content: center;
+        align-items: center;
+        gap: var(--boxel-sp-xxs);
+        height: 100%;
+      }
+      .title {
+        overflow: hidden;
+        text-overflow: ellipsis;
+        -webkit-box-orient: vertical;
+        -webkit-line-clamp: 2;
```
Comment (jurgenwerk) [inline packages/experiments-realm/person.gts:45]: Line clamping won't work without
`display: -webkit-box;`( https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-line-clamp)

**Comment by lukemelia** [comment]
The BlogPost should probably use markdown to render the body

**Comment by lukemelia** [comment]
For Pet, let's try displaying the metadata similar to how we did with Media Registry:

image

But with actual values instead of "N/A"

**Comment by FadhlanR** [comment]
Latest screenshots

https://github.com/user-attachments/assets/76e4d4b5-cea4-469d-9fda-96418a5a31c9


https://github.com/user-attachments/assets/3400b183-9523-48a4-bdc1-5e0191c062e6


https://github.com/user-attachments/assets/31ca6d82-c2e7-44b1-9d8e-6b3cf074af49

**Review by jurgenwerk — APPROVED** [review]

**Comment by lukemelia** [comment]
The Pet name value pairs should never have the horizontal scroll, IMO. The task of authoring a fitted component is to show as much important info as you can fit and leave out that which doesn't make the cut. Using a horizontal scroll defeats that intention.

In the Person fitted format, when we do the delegated render to the Pet, we should probably make that @format="fitted" as well since we have a limited space we are working within.

**Pull request merged**